src/common/gui/builtinThemes.ts

```diff
@@ -98,90 +98,65 @@ const getLogo = (isDark: boolean, isDefault: boolean) => {
 	return isDarkOrDefault ? getMailLogoSvg(logo_text_bright_grey, logo_text_bright_grey, logo_text_bright_grey) : getMailLogoSvg(red, secondary_red, black)
 }
 
+/**
+ * Color token definitions for each built-in theme.
+ * We follow the color roles from Material 3 to define the names of the tokens,
+ * the following link might be helpful to know which color token should be used in which situation.
+ * https://m3.material.io/styles/color/roles
+ *
+ * When a new color role is needed, please follow the instructions below.
+ * https://m3.material.io/styles/color/advanced/define-new-colors#baed14ce-4be8-46aa-8223-ace5d45af005
+ */
 export const themes = (): Themes => {
 	const isCalendarApp = client.isCalendarApp()
 	const lightRed = Object.freeze({
 		themeId: !isCalendarApp ? "light" : "light_secondary",
 		logo: getLogo(false, !isCalendarApp),
-		button_bubble_bg: grey_lighter_3,
-		button_bubble_fg: grey_darker_1,
-		content_fg: grey_darker_1,
-		content_button: grey_darker_0,
-		content_button_selected: red,
-		content_button_icon: light_white,
-		content_button_icon_selected: light_white,
-		content_accent: red,
+		// Campaign colors
+		tuta_color_nota: red_nota,
 		content_accent_tuta_bday: dark_purple,
 		content_accent_secondary_tuta_bday: light_purple,
-		content_bg: light_white,
 		content_bg_tuta_bday: dark,
-		content_border: grey_lighter_1,
-		content_message_bg: grey_lighter_0,
-		header_bg: light_white,
-		header_box_shadow_bg: grey_lighter_1,
-		header_button: grey_darker_0,
-		header_button_selected: red,
-		list_bg: light_white,
-		list_alternate_bg: grey_lighter_4,
-		list_accent_fg: red,
-		list_message_bg: grey_lighter_0,
-		list_border: grey_lighter_2,
-		modal_bg: grey_darker_1,
-		elevated_bg: light_white,
-		navigation_bg: grey_lighter_4,
-		navigation_border: grey_lighter_2,
-		navigation_button: grey_darker_0,
-		navigation_button_icon: light_white,
-		navigation_button_selected: red,
-		navigation_button_icon_selected: light_white,
-		navigation_menu_bg: grey_lighter_3,
-		navigation_menu_icon: grey,
-		error_color: SONNE,
-		tuta_color_nota: red_nota,
+		// Basic color tokens
+		primary: red,
+		on_primary: light_white,
+		secondary: grey_lighter_3,
+		on_secondary: grey_darker_1,
+		error: SONNE,
+		surface: light_white,
+		surface_container: grey_lighter_4,
+		on_surface_fade: grey_lighter_0,
+		on_surface: grey_darker_1,
+		on_surface_variant: grey_darker_0,
+		outline: grey_lighter_1,
+		outline_variant: grey_lighter_3,
+		scrim: grey_darker_1,
```

Comment:
same for `scrim`, can't find reference for it

Reply:
oh I guess it's this one?
https://m3.material.io/styles/elevation/applying-elevation#92b9fb39-f0c4-4829-8e4d-97ac512976aa